hoomd/md/pytest/test_alj.py

```diff
@@ -104,7 +109,7 @@ def test_conservation(simulation_factory, lattice_snapshot_factory):
     velocities = []
     total_energies = []
     while sim.timestep < n_total:
-        sim.run(1_000)
+        sim.run(1000)
```

Comment:
Why the change?

Reply:
Isn't the above statement syntactically incorrect?

Reply:
No, Python allows you to use underscores as separators for readability in numbers as of Python 3.6 (specifically, [PEP 515](https://www.python.org/dev/peps/pep-0515/)).